srcpkgs/trash-cli/template

```diff
@@ -1,14 +1,16 @@
 # Template file for 'trash-cli'
 pkgname=trash-cli
-version=0.20.12.26
-revision=2
+version=0.22.4.16
+revision=1
 build_style=python3-module
-hostmakedepends="python3"
+hostmakedepends="python3 python3-setuptools"
 depends="python3-psutil"
 checkdepends="$depends python3-pytest python3-six python3-mock"
 short_desc="Command line interface to the freedesktop.org trashcan"
 maintainer="Anjandev Momi <anjan@momi.ca>"
 license="GPL-2.0-or-later"
 homepage="https://github.com/andreafrancia/trash-cli"
 distfiles="https://github.com/andreafrancia/trash-cli/archive/${version}.tar.gz"
-checksum=06173eaca9b7fbda661ac7d84f900c7f4d17db5dd6ee4b3617bcf2182cde0f31
+checksum=6f2d306abbb0cb857d6fc6697a65e006758458ad85793cc0b62f64be993f9c68
+# https://github.com/void-linux/void-packages/runs/6644653643
```

Comment:
afaik ci runs are only available for a very short time, can we mention the reason here?

Reply:
what is the reason?

Reply:
From what I remember, CI logs expire and won't be available in a few weeks.

Reply:
sorry I mean't whats the reason for the failure?

Reply:
```
E       Expected:"[Errno 13] Permission denied: '/tmp/tmpc5nl11zm_trash_cli_test/Trash/info/unreadable.trashinfo'\n"
E         Actual:'Parse Error: /tmp/tmpc5nl11zm_trash_cli_test/Trash/info/unreadable.trashinfo: Unable to parse Path.\n'
```
Seems like the test wasn't prepared to be run by `root`

Reply:
Okay I changed the comment